test/cqlpy/run.py

```
@@ -562,3 +562,23 @@ def setup_ssl_certificate(dir):
+# Set up mTLS (mutual TLS) certificates for testing client certificate
+# authentication. Creates:
+#   dir/ca.key, dir/ca.crt    - a self-signed CA certificate
+#   dir/client.key, dir/client.crt - a client certificate with CN "cassandra",
+#                                    signed by the above CA
+# The CA certificate (dir/ca.crt) should be passed to Scylla as the truststore
+# so that Scylla can verify client certificates. The client key and certificate
+# can be used by test clients to authenticate themselves.
+def setup_mtls_certificate(dir):
+    # FIXME: error checking (if "openssl" isn't found, for example)
+    # Create a self-signed CA certificate
+    os.system(f'openssl genrsa 2048 > "{dir}/ca.key"')
```

Ruff (0.16.1)

[error] Starting a process with a shell, possible injection detected

(S605)